### DISCUSSION TITLE
Add scroll to bottom handler for combobox

### DISCUSSION THREAD

**Discussion by wghglory** [vmware/clarity#6078] [Ideas]
We have lots of data for combobox, so we'd like to load new data when scrolling down to the dropdown bottom. But there isn't any scrollToBottom handler. 

https://ngx.eng.vmware.com/@vmw/ngx-components/combobox/demo has this feature, but a11y is not working well.

**Comment by mathisscott** [comment]
We currently do not have any load-on-scroll or "infinite scroll" features in Clarity. The blocker on that is accessibility – for us. We made several attempts at different patterns but, to date, no pattern has met accessibility requirements.

As we move away from the pure angular components towards the web components in `@cds/core`, our efforts to add new features to the legacy angular components has begun winding down. This means that this kind of feature will not be added to the `@clr/angular` combobox.

That said, the Clarity Core components are much more flexible and easier to customize. So while I can't say, at this time, if Clarity Core components will feature infinite scroll capabilities, I _can_ say that customizing an existing Clarity Core component to have that functionality would be straightforward. But any team implementing a customization like that would own any accessibility issues that arise.

Thanks for reaching out.
